src/components/components/ebay-dialog-base/component.js

```diff
@@ -23,10 +23,14 @@ module.exports = {
         this.clickTarget = el;
     },
 
-    getActiveElement() {
+    getActiveElement(input) {
+        let closeFocusEl;
+        if (input && input.closeFocus) {
+            closeFocusEl = document.getElementById(input.closeFocus);
```

Comment:
`document.querySelector` would be more flexible than `document.getElementById`

Reply:
Yes, possibly.
However this is in line with `focus` which uses `document.getElementById`. For consistency this is why it is done that way. 
Plus we don't want to deal with conflicts, if there are multiple matches on the page that can lead to other errors. Forcing ID makes it much cleaner and easier.

However, in the next major version we can revisit this and make both support `querySelector` if necessary. But we definitely want to keep this in sync with `focus`